cli/src/pcluster/config/update_policy.py

```diff
@@ -116,7 +116,8 @@ def __eq__(self, other):
         "modify-ebs-volume",
     ),
     "pcluster_stop": lambda change, patch: (
-        f"Stop the cluster with the following command:\npcluster stop {patch.cluster_name}"
+        "Stop the cluster with the following command:\npcluster update-compute-fleet --cluster-name  "
```

Comment:
I'd say stop the compute fleet. 

Then is STOP_REQUESTED valid for both Slurm and Batch? If it's not the case we can just say stop the compute fleet, without mentioning the command.

Reply:
> is STOP_REQUESTED valid for both Slurm and Batch?

I assume so, based on the fact that `UpdatePolicy.AWSBATCH_CE_MAX_RESIZE` has uses the same reason.

Reply:
Unfortunately it is not true: https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/api/controllers/cluster_compute_fleet_controller.py#L66

Reply:
so does it need to be DISABLED instead of STOP_REQUESTED for batch?

Reply:
> without mentioning the command

i think not including  the command to run defeats the purpose of providing suggested actions